paragraph/scheduling/graph_scheduler.h

```diff
@@ -39,11 +39,16 @@ class GraphScheduler {
   ~GraphScheduler() = default;
 
   // Creates a new scheduler and verifies the that graph is ready to be
-  // scheduled
+  // scheduled.
   // Part of Public API with simulators
   static shim::StatusOr<std::unique_ptr<GraphScheduler>> Create(
       Graph* graph);
 
+  // Initializes graph execution and sets time when available instructions are
+  // ready. Can be performed much later after scheduler creation.
+  // Part of Public API with simulators
+  absl::Status Init(double seconds = 0.0);
```

Comment:
I don't think this should have a default.

Reply:
done